shared/actions/chat/index.js

```diff
@@ -1016,6 +1016,33 @@ function* _exitSearch() {
   yield put(Creators.setInboxFilter([]))
 }
 
+function* _selectSearchResultWithMovement({payload: {movement}}: Constants.SelectSearchResultIdWithMovement) {
+  const searchResults = yield select(({chat: {searchResults}}: TypedState) => searchResults)
+  const selectedSearchId = yield select(({chat: {selectedSearchId}}: TypedState) => selectedSearchId)
+  const index = searchResults.indexOf(selectedSearchId)
```

Comment:
What if for some reason this returns `-1`? Should there be a warning / reset logic?